test/test_keeps_files_on_local_machine.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -e
+
+# You can run it from any directory.
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# Execute common pre-setup, include test functions.
+source "$DIR/common.sh"
+
+printTestStarted
+
+# Create several files that should be synced to remote machine AND KEEPT after execution.
+mkdir "$BUILD_DIR/src"
+touch "$BUILD_DIR/src/file1.txt"
+touch "$BUILD_DIR/src/file2.txt"
+touch "$BUILD_DIR/src/file3.txt"
```

Comment:
Sure, but I didn't want to complicate source code, let's leave as is for easier readability by us and others?